File: row-change-event-channel/src/test/scala/nl/haploid/event/channel/DockerConfiguration.scala
package nl.haploid.event.channel

import com.github.dockerjava.api.DockerClient
import com.github.dockerjava.core.DockerClientBuilder
import org.springframework.beans.factory.annotation.Value
import org.springframework.context.annotation.{Bean, Configuration}

@Configuration
class DockerConfiguration {

  @Value("${DOCKER_HOST}") val host: String = null

  @Bean
  def docker: DockerClient = DockerClientBuilder.getInstance.build
}

File: row-change-event-channel/src/test/java/nl/haploid/event/channel/DockerConfiguration.java
package nl.haploid.event.channel;

import com.github.dockerjava.api.DockerClient;
import com.github.dockerjava.core.DockerClientBuilder;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class DockerConfiguration {

    @Bean
    public DockerClient docker() {
        return DockerClientBuilder.getInstance().build();
    }
}
